src/components/Layout/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Head from 'next/head';
+
+import styles from './index.module.css';
+
+interface LayoutProps {
+  children: React.ReactNode;
+}
+
+const Layout = ({ children }: LayoutProps) => {
+  return (
+    <>
+      <Head>
+        <title>TIL</title>
+        <meta name="description" content="What Nimble has learned today" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main className={styles.mainContainer}>{children}</main>
```

Comment:
I am just curious why we are not applying the utility classes directly here? Using the `@apply` directive requires us to come up with class names which tailwind [discourages](https://tailwindcss.com/docs/utility-first) firmly?